docs/api/CallComposite/classes/LocalizationConfiguration.md

```diff
@@ -16,26 +16,48 @@ Creates an instance of `LocalizationConfiguration` with related parameters.
 
 ```java
 public LocalizationConfiguration(
-    final LanguageCode languageCode
+    final Locale locale
```

Comment:
why this PR has changes under LocalizationConfiguration file?

Reply:
This change came with merge from develop/beta